subprojects/docs/src/snippets/testing/test-suite-multi-configure-each/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+import org.gradle.api.internal.tasks.testing.junit.JUnitTestFramework
+
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    `java-library`
+}
+
+version = "1.0.0"
+group = "org.gradle.sample"
+
+repositories {
+    mavenCentral()
+}
+
+// tag::multi-configure[]
+testing {
+    suites.configureEach { // <1>
+        if (this is JvmTestSuite) {
```

Comment:
You could do this with `suites.withType<JvmTestSuite>().configureEach {` I think.

Reply:
I don't know about this one.  Isn't the original code is a slightly better translation of the Groovy code?  It should be obvious from the next example that you can restrict the set of suites you are iterating over by adding the `withType` call prior to iterating.  This also lets the complexity build up gradually over the examples.

I think I'd rather keep it as is here.